packages/now-cli/test/integration.js

```diff
@@ -2237,6 +2237,40 @@ test('ensure `github` and `scope` are not sent to the API', async t => {
   t.is(output.exitCode, 0, formatOutput(output));
 });
 
+test.only('deploy with unknown `NOW_ORG_ID` and `NOW_PROJECT_ID`', async t => {
```

Comment:
```suggestion
test('deploy with unknown `NOW_ORG_ID` and `NOW_PROJECT_ID`', async t => {
```

Reply:
I updated around the same time you commented 😅